src/screens/ManageDetailsSessions/ManageDetailsSessions.js

```diff
@@ -212,7 +214,10 @@ class ManageDetailsSessions extends React.Component<Props, State> {
   }
 }
 
-const mapStateToProps = ({ user: { data: user }, walletConnect: { connectors, requests } }) => ({
+const mapStateToProps = ({
+  user: { data: user },
+  walletConnect: { connectors, requests },
+}) => ({
```

Comment:
To continue on this change we can add Reducer type here.

Reply:
Good suggestion!